hydra-test-utils/test/Test/HSpec/MarkdownFormatterSpec.hs

```diff
@@ -59,7 +60,7 @@ hspecWithMarkdown title tmpDir aSpec = do
         , configFormat = Just (markdownFormatter title markdownFile)
         }
       aSpec
-  readFile markdownFile
+  unpack <$> readFileBS markdownFile
```

Comment:
I would not use `Char8.unpack`, but rather `decodeUtf8` to properly support unicode symbols in markdown.

Reply:
We have this in scope: https://hackage.haskell.org/package/relude-1.1.0.0/docs/Relude-String-Conversion.html#v:decodeUtf8

Reply:
Didn't knew about this one, thanks!